.github/push_ghpages.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+# please set the $GH_TOKEN in your travis dashboard
+
+if [ "$TRAVIS_BRANCH" = "master" ] || [ "$TRAVIS_BRANCH" = "develop" ] && [ "$TRAVIS_REPO_SLUG" = "flyve-mdm/glpi-plugin" ]; then
```

Comment:
If we rename (again) the repo, the current code will break.

I think the script will be stronger if you use the env var TRAVIS_PULL_REQUEST instead of TRAVIS_REPO_SLUG. 

see https://docs.travis-ci.com/user/environment-variables/

Reply:
Ok it cloud work but I have a doubt. Doesn't that make the forks execute the scripts if they also have PR?

Reply:
No. As far as I understand, this will execute the script if there is no PR. If you fork the project and push a commit on master or develop branches of this fork, the build will execute the script.

This requires the forked project is properly configured : it requires a GH token, and the repo to update is forked as well. Maybe your script needs an update to write on the good organization. 

Having a build script which runs no matter the repo / orga changes looks good to me. Less hassle if we change anything.